service/src/main/java/bio/terra/workspace/amalgam/landingzone/azure/LandingZoneApiDispatch.java

```diff
@@ -239,13 +239,15 @@ private ApiAzureLandingZoneDeployedResource toApiAzureLandingZoneDeployedResourc
       return new ApiAzureLandingZoneDeployedResource()
           .resourceId(resource.resourceId())
           .resourceType(resource.resourceType())
+          .tags(resource.tags())
           .region(resource.region());
     }
     if (purpose.getClass().equals(SubnetResourcePurpose.class)) {
       return new ApiAzureLandingZoneDeployedResource()
           .resourceParentId(resource.resourceParentId().orElse(null)) // Only available for subnets
           .resourceName(resource.resourceName().orElse(null)) // Only available for subnets
           .resourceType(resource.resourceType())
+          .tags(resource.tags())
```

Comment:
the `LandingZoneResource` model already had `tags`; this change copies those tags from the `LandingZoneResource` to the `ApiAzureLandingZoneDeployedResource`.